solutions/swb-reference/src/SWBStack.ts

```diff
@@ -117,11 +117,22 @@ export class SWBStack extends Stack {
       ALB_INTERNET_FACING
     } = getConstants();
 
-    super(app, STACK_NAME, {
-      env: {
-        region: AWS_REGION
-      }
-    });
+    const MAIN_ACCT_ID: string = process.env.CDK_DEFAULT_ACCOUNT || '';
+
+    if (MAIN_ACCT_ID) {
```

Comment:
do we need to do an `if` here? would `account: process.env.CDK_DEFAULT_ACCOUNT || undefined` work?

Reply:
Good call, fixed.